src/action.ts

```diff
@@ -12,6 +12,22 @@ export namespace Action {
       core.info(`action: ${action}`)
       core.info(`event: ${Util.getEventName()}`)
 
+      const startDate = Util.getStartDate()
+      const endDate = Util.getEndDate()
+      if (startDate || endDate) {
+        core.info(`StartDate: ${startDate} - EndDate ${endDate}`)
+        const now = new Date()
+        core.info(`Today: ${now}`)
+
+        if (!(startDate && startDate < now)) {
+          throw new Error('Running before StartDate - not executing')
```

Comment:
I'm not sure if throwing an error here (and below) is desired behavior (do people monitor workflow executions)?

Happy to refactor this to something better if that's the preferred approach.